app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1><%= t ".title" %></h1>
+<p>
+  <%= t ".content" %>
+  <a href="https://www.railstutorial.org/book"><em>Ruby on Rails Tutorial</em></a>.
```

Comment:
i18n em nhé

Reply:
chỗ này em dùng i18n rồi truyền param vào nha, 1 key content chứa toàn bộ nội dung

Reply:
khi dùng i18n nó sẽ bị xuất ra dạng text, trong đó chứa cả thẻ html thi em dùng html_safe để escape nó ra html nhé

Reply:
những chỗ khác, tương tự em nhé, fix hết đi giúp anh